src/screens/Exchange/Exchange/Exchange.js

```diff
@@ -83,14 +82,20 @@ function Exchange() {
   const [fromAmount]: [string] = useDebounce(rawFromAmount, 500);
 
   const chainConfig = useChainConfig(CHAIN.ETHEREUM);
+  const chainSupportedAssets = useChainSupportedAssets(CHAIN.ETHEREUM);
```

Comment:
```suggestion
  const supportedAssets = useChainSupportedAssets(CHAIN.ETHEREUM);
```

Reply:
I would either leave as `chainSupportedAssets` or rename to `ethereumSupportedAssets`, wdyt?

Reply:
ethereumSupportedAssets 👍🏻